DNN Platform/Modules/DnnExportImport/Components/Dto/TriCheckedState.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace Dnn.ExportImport.Components.Dto
+{
+    /// <summary>Three check states.</summary>
+    public enum TriCheckedState
+    {
+#if false

```

Comment:
Hmm, I am not familiar with this pattern, I am fine leaving it as is, but anyone understands this purpose ? `#if false` to me looks like that code will never get hit no ?

Reply:
Correct, I think it's just a fancy way of commenting out the code.  it's just there to show what the previous version of the type was.  We should be fine to remove it, if we want.